### PR TITLE
Prevent caching of the status end-point

### PR DESCRIPTION
**Pull request by jon-betts**
For: hypothesis/product-backlog#1198

This also includes a fix that allows the func tests to be run individually.

### CODE REVIEW

File: via/app.py [hunk 1]
```diff
@@ -29,19 +34,19 @@ def load_settings(settings):
...
+
+        if value is None and options.get("required"):
             raise ValueError(f"Param {param} must be provided.")
 
     # Configure sentry
     settings["h_pyramid_sentry.filters"] = SENTRY_FILTERS
 
-    settings["checkmate_ignore_reasons"] = os.environ.get("CHECKMATE_IGNORE_REASONS")
-    settings["signed_urls_required"] = asbool(os.environ.get("SIGNED_URLS_REQUIRED"))
-    settings["nginx_secure_link_secret"] = os.environ["NGINX_SECURE_LINK_SECRET"]
```
Comment (jon-betts) [inline via/app.py:43]: These settings were added outside of the mechanism above, meaning these ones didn't respect things passed in via `settings`

**Comment by jon-betts** [comment]
Fixing up the settings in the func tests has broken something the tests were relying to pass for now... not a Friday job though

File: tests/conftest.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def pyramid_settings():
         "checkmate_url": "http://localhost:9099",
         "nginx_secure_link_secret": "not_a_secret",
         "via_secret": "not_a_secret",
-        "signed_urls_required": True,
+        "signed_urls_required": False,
```
Comment (jon-betts) [inline tests/conftest.py:15]: Yeah, turns out this isn't actually the value we were using! This was being ignored and overwritten with the empty env var.

File: tox.ini [hunk 3]
```diff
@@ -28,7 +28,7 @@ setenv =
     dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
     dev: CHECKMATE_URL = http://localhost:9099
     dev: CHECKMATE_IGNORE_REASONS = {env:CHECKMATE_IGNORE_REASONS:publisher-blocked}
-    {dev,functests}: NGINX_SECURE_LINK_SECRET = not_a_secret
+    dev: NGINX_SECURE_LINK_SECRET = not_a_secret
```
Comment (jon-betts) [inline tox.ini:31]: This was the only value being set from the outside intentionally. Now it's all in the `conftest.py`

**Review by seanh — APPROVED** [review]

**Pull request merged**